clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -176,20 +176,32 @@ bool
 check_branch_target_after_branch()
```

Comment:
> i#6345: Allow CPU ID marker between a branch and it's target

grammar: s/it's/its/

Reply:
I've updated the description to reflect the new changes.